### PR TITLE
Added 2 new blender nodes and fixed some Godot shader issues

### PR DESCRIPTION
**Pull request by wagnerfs**
Added HueSaturation and Invert (only for color input) nodes
Fix normal mapping node to Godot shader
Fix albedo textures not receiving hint_albedo in Godot

### CODE REVIEW

File: io_scene_godot/converters/material/script_shader/shader_functions.py [hunk 1]
```diff
@@ -245,6 +245,41 @@ def __init__(self, code, input_sockets, output_properties):
 void node_combine_rgb(float r, float g, float b, out vec4 color) {
     color = vec4(r, g, b, 1.0);
 }
+"""),
+
+	ShaderFunction(code="""
+void node_hsv(float fac, float in_hue, float in_saturation, float in_value,
```
Comment (Jason0214) [inline io_scene_godot/converters/material/script_shader/shader_functions.py:251]: The added code here can be aligned better :)

**Review by Jason0214 — APPROVED** [review]
Approve on fixing the alignments.

Looks fine to me. But apologize that I am no longer have the context to give any opinion on the correctness of the fix or the new added node.
I think it can be merged if there is no issue raised by other people.

File: io_scene_godot/converters/material/script_shader/shader_functions.py (hunk 1 above)
Comment (wagnerfs) [reply io_scene_godot/converters/material/script_shader/shader_functions.py:251]: I honestly just followed the rest of the code's design, thought any additional code should look like that

Comment (Jason0214) [reply io_scene_godot/converters/material/script_shader/shader_functions.py:251]: Forgive my English, what I really mean is "indentations".
From my browser, I am seeing some inconsistent indentations
image

However I just found when I copy this piece of code into my editor, they are all four spaces indented. Honestly I don't know what's happening.  One thing for sure is there are some extra spaces can be removed at line 262, line 266, and line 270

Comment (Williangalvani) [reply io_scene_godot/converters/material/script_shader/shader_functions.py:251]: I believe github shows Tabs as 8 spaces, while most editors show it as 4.

Comment (Calinou) [reply io_scene_godot/converters/material/script_shader/shader_functions.py:251]: > I believe github shows Tabs as 8 spaces, while most editors show it as 4.

Note that you can install the [Refined GitHub](https://github.com/sindresorhus/refined-github) browser add-on to change this :slightly_smiling_face:

Comment (wagnerfs) [reply io_scene_godot/converters/material/script_shader/shader_functions.py:251]: Oh okay, I know what the problem is, those extra lines were tabs, but blender expects everything to be spaces, I converted TABs into spaces on node_convertes.py but not on shader_functions.py, gonna fix it real quick

**Pull request merged**